scipy/special/orthogonal.py

```diff
@@ -1033,11 +1070,11 @@ def _newton(n, x_initial, maxit=5):
 def _roots_hermite_asy(n):
     r"""Gauss-Hermite (physicst's) quadrature for large n.
 
-    Computes the sample points and weights for Gauss-Hermite quadrature.
-    The sample points are the roots of the n-th degree Hermite polynomial,
+    Computes the sample points and weights for GaussHermite quadrature.
+    The sample points are the roots of the nth degree Hermite polynomial,
     :math:`H_n(x)`.  These sample points and weights correctly integrate
-    polynomials of degree :math:`2n - 1` or less over the interval
-    :math:`[-\infty, \infty]` with weight function :math:`f(x) = e^{-x^2}`.
+    polynomials of degree :math:`2n  1` or less over the interval
```

Comment:
what's going on with removing all `-`'s in this docstring?

Reply:
Whoa, I have no idea what happened there! Let me fix that...